fr/tutorials-and-examples/blog-auth-example/auth.rst

```diff
@@ -99,10 +99,10 @@ classe obtenue grâce à l'utilitaire de génération de code fourni par CakePHP
             if ($this->request->is('post')) {
                 $user = $this->Users->patchEntity($user, $this->request->getData());
                 if ($this->Users->save($user)) {
-                    $this->Flash->success(__("L'utilisateur a été sauvegardé."));
+                    $this->Flash->success(__("L\'utilisateur a été sauvegardé."));
```

Comment:
Quotes only need to be escaped inside of quotes of the same type, ie single quotes inside of single quotes, and double quotes inside of double quotes.

Reply:
@AlexDev4 Can you correct this so we can merge?

Reply:
Sorry, I should've probably mentioned it initially, rather than expecting that everyone would figure that I've only picked this line as an example, but there's more occurrences of the same problem in the changes.